tests/macsec/test_neighbor_reboot.py

```diff
@@ -0,0 +1,42 @@
+'''
+
+The test case will verify a neighbor reboot while Macsec is Active.
+
+Step 1: Configure Macsec between neighbor and DUT
+Step 2: Save config on neighbor and reboot
+Step 3: Verify macsec connection is re-established after reboot
+
+'''
+
+from time import sleep
+import pytest
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.macsec_required,
+    pytest.mark.topology("t2")
+]
+
+
+def test_neighbor_reboot(duthost, nbrhosts, request, enum_rand_one_frontend_asic_index):
+    if request.config.getoption("neighbor_type") != "sonic":
+        pytest.skip("Neighbor type must be sonic")
+    asic_index = enum_rand_one_frontend_asic_index
+    namespace = duthost.get_namespace_from_asic_id(asic_index)
+    dut_lldp_table = duthost.shell("show lldp table")['stdout'].split("\n")[3].split()
+    dut_to_tor1_int = dut_lldp_table[0]
+    tor1 = dut_lldp_table[1]
```

Comment:
I think here `tor1` was not limited to tor, it could be T2/T3, right? could you update the name to avoid misleading

Reply:
@wenyiz2021 I changed the name of the variable.